### ISSUE TITLE
fails to build with either-5

### ISSUE THREAD

**Issue by juhp** [MichaelXavier/drifter-postgresql#7]
For Stackage Nightly:

```
Building library for drifter-postgresql-0.1.0..
[1 of 1] Compiling Drifter.PostgreSQL ( src/Drifter/PostgreSQL.hs, dist/build/Drifter/PostgreSQL.o )

src/Drifter/PostgreSQL.hs:23:1: error:
    Could not find module ‘Control.Monad.Trans.Either’
    Perhaps you meant
      Control.Monad.Trans (from mtl-2.2.1)
      Control.Monad.Writer (from mtl-2.2.1)
    Use -v to see a list of the files searched for.
   |
23 | import           Control.Monad.Trans.Either
```

**Comment by MichaelXavier** [comment]
This is fixed at drifter-postgresql 0.2.0

**Comment by juhp** [comment]
Thanks
